openmp/libomptarget/test/api/omp_get_num_procs.c

```diff
@@ -0,0 +1,15 @@
+// RUN: %libomptarget-compile-run-and-check-generic
```

Comment:
You might want to require certain targets otherwise the test will fail since by default it returns 0.